job-descriptions/fullstack-engineer.md

```diff
@@ -0,0 +1,52 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Full Stack Engineer
+Sourcegraph is seeking an engineer that thrives on owning big problems across domains and different levels of the stack — you grok how the ultimate measure of your work is how users experience the work you do, and that means you wear any hat necessary to that end, from product, to developer, and everything in between. You are a polyglot, quick learner and fearless into diving into unknown areas of the systems you work with. You believe that communicating clearly and empathetically and your relationships with others is critical to our success.
+## Qualifications
+
+We are looking for a full stack engineer who has strong fundamentals in good software development techniques, design patterns and best practices. In your career you have worked with and refactored existing code bases. You are both productive and pragmatic because you believe software is only useful if it is used. Collaborating with small high performing teams, you have built and deployed production-ready software that delivers value to customers.
+* Strong working knowledge with API design and architecture.
+* Skilled at building and testing end-to-end applications (e.g., unit testing, integration and e2e testing).
+* Good knowledge of Git in particular and other version control systems in general.
+* Single page applications or progressive web apps and Web APIs.
```

Comment:
The others are statements of wants, whereas this is a thing.  For example "Good knowledge of git..."; perhaps:

Experience developing single page applications or progressive web applications and APIs.